test.cpp

```
@@ -0,0 +1,48 @@
+    Circle() {}
+    void draw() { std::cout << "circle" << std::endl; }
+};
+
+void nullDeref(int* p) {
+    if (p == NULL) {
+        *p = 5;
+    }
+}
+
+int divide(int numerator, int denominator) {
+    return numerator / denominator;
```

Cppcheck (2.20.0)

[error] Division by zero.

(zerodiv)